<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientDialog</class>
 <widget class="QDialog" name="AddClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="lblCuit">
     <property name="text">
      <string>CUIT:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="txtName"/>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblEmail">
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLineEdit" name="txtEmail"/>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QLineEdit" name="txtLocation"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPhone">
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblLocation">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="txtPhone"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="txtCuit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtCuit</tabstop>
  <tabstop>txtPhone</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtLocation</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
